lib/assessment/label.py

```diff
@@ -104,20 +109,42 @@ def bedrock_anthropic_label_student_work(self, prompt, rubric, student_code, stu
             raise Exception(f"Error parsing llm_model: {llm_model} bedrock_model: {bedrock_model}")
 
         anthropic_prompt = self.compute_anthropic_prompt(prompt, rubric, student_code, examples=examples)
-        body = json.dumps({
-            "prompt": anthropic_prompt,
-            "max_tokens_to_sample": 4000,
-            "temperature": temperature,
-            # "top_p": 0.9,
-        })
+        if "claude-3" in bedrock_model:
+            body = json.dumps({"anthropic_version": "bedrock-2023-05-31",
+                               "messages": [{"role": "user",
+                                             "content": [{"type": "text",
+                                                          "text": anthropic_prompt}
+                                                        ]
+                                            }]
+                              })
+        else:
+            body = json.dumps({
+                "prompt": anthropic_prompt,
+                "max_tokens_to_sample": 4000,
+                "temperature": temperature,
+                # "top_p": 0.9,
+            })
         accept = 'application/json'
         content_type = 'application/json'
         response = bedrock.invoke_model(body=body, modelId=bedrock_model, accept=accept, contentType=content_type)
```

Comment:
per the notes in the jira ticket, the timeout here should be 150 seconds. this can be done in a next PR if needed, but should be done before we launch the claude release. some reasoning here: https://github.com/code-dot-org/code-dot-org/pull/58411 --> "timeouts" section in PR description